sycl/test/check_device_code/fpga_ihs_float.cpp

```diff
@@ -19,431 +19,387 @@ constexpr bool ToSign = true;
 constexpr bool SignOfB = false;
 
 template <int EA, int MA, int Eout, int Mout>
-void ap_float_cast() {
+SYCL_EXTERNAL auto ap_float_cast() {
   sycl::detail::ap_int<1 + EA + MA> A;
-  sycl::detail::ap_int<1 + Eout + Mout> float_cast_res =
-      __spirv_ArbitraryFloatCastINTEL<1 + EA + MA, 1 + Eout + Mout>(
-          A, MA, Mout, Subnorm, RndMode, RndAcc);
-  // CHECK: call spir_func i40 @_Z{{[0-9]+}}__spirv_ArbitraryFloatCastINTEL{{.*}}(i40 {{[%a-z0-9.]+}}, i32 28, i32 30, i32 0, i32 2, i32 1)
+  return __spirv_ArbitraryFloatCastINTEL<1 + EA + MA, 1 + Eout + Mout>(
```

Comment:
I' d move local variables to the function arguments.

Reply:
Fixed!